internal/uvm/wait.go

```diff
@@ -1,6 +1,19 @@
 package uvm
 
+import (
+	"github.com/Microsoft/hcsshim/internal/logfields"
+	"github.com/sirupsen/logrus"
+)
+
 // Waits synchronously waits for a utility VM to terminate.
 func (uvm *UtilityVM) Wait() error {
-	return uvm.hcsSystem.Wait()
+	err := uvm.hcsSystem.Wait()
+
+	logrus.WithField(logfields.ContainerID, uvm.ID).
```

Comment:
You meant `logfields.UVMID`

Reply:
I just noticed that I used ContainerID because that's what's used by internal\hcs\system.go. Meaning today we have the begin/end operation log messages logging with cid=<uvm id> for UVMs. We might want to look at fixing this in the future.

Reply:
ERRO[0001] hcsshim::ComputeSystem - End Operation - Error  cid=uvmboot-0 error="Start uvmboot-0: The virtual machine or container exited unexpectedly." hcs-op="hcsshim::ComputeSystem::Start"